index.js

```diff
@@ -0,0 +1,14 @@
+// First function should be a regular function to divide two numbers.
+
+function divide (num1, num2){
+        return num1 / num2;
+}
```

Comment:
This can also be an arrow function 😺 
```suggestion
const divide = (num1, num2) => num1 / num2
```

Reply:
Yeah, the tests seem to want to first function to be a regular function. It was confusing. Like I originally wrote it as an arrow function but rewrote it when reading the tests.

Reply:
Gotchya, yeah makes sense!